audinterface/core/feature.py

```diff
@@ -77,6 +77,18 @@ class Feature:
             Afterwards processing is applied to each segment
         keep_nat: if the end of segment is set to ``NaT`` do not replace
             with file duration in the result
+        min_signal_length: minimum signal length
+            required by ``process_func``.
+            If value is as a float or integer
```

Comment:
A string, e.g. '1s'  is also supported maybe we should mention that?

Reply:
I have now adjusted the docstring to be the same as for `start` or `win_dur`